src/tests/Common/Coreclr.TestWrapper/CoreclrTestWrapperLib.cs

```diff
@@ -305,85 +305,99 @@ public int RunTest(string executable, string outputFile, string errorFile, strin
             using (var errorWriter = new StreamWriter(errorStream))
             using (Process process = new Process())
             {
-                // Windows can run the executable implicitly
-                if (OperatingSystem.IsWindows())
+                if (File.Exists($"{testBinaryBase}/.retry"))
                 {
-                    process.StartInfo.FileName = executable;
+                    // We have requested a work item retry because of an infra issue - no point executing further tests
+                    outputWriter.WriteLine("\nretry file was found, quit testing...");
```

Comment:
I like the comment more than the message :)

```suggestion
                    outputWriter.WriteLine("\nWork item retry had been requested earlier - skipping test...");
```